master/buildbot/test/util/steps.py

```diff
@@ -51,9 +51,12 @@ def _dict_diff(d1, d2):
 
     for k in both:
         if isinstance(d1[k], dict) and isinstance(d2[k], dict):
-            missing_in_v1, missing_in_v2, different_in_v = _dict_diff(d1[k], d2[k])
-            missing_in_d1.extend(['{0}.{1}'.format(k, m) for m in missing_in_v1])
-            missing_in_d2.extend(['{0}.{1}'.format(k, m) for m in missing_in_v2])
+            missing_in_v1, missing_in_v2, different_in_v = _dict_diff(
```

Comment:
mmhh very weird. I think you have a wrong autopep8 version or badly configured editor?

the autopep8 from the hooks is enforcing 100 columns, not 72.
I just verified that I can modify that file, and autopep8 wont remove the spaces between = nor reduce lines